examples/training/classification/imagenet/basic_training.py

```diff
@@ -68,6 +69,11 @@
 flags.DEFINE_boolean(
     "use_xla", True, "Whether or not to use XLA (jit_compile) for training."
 )
+flags.DEFINE_boolean(
+    "use_mixed_precision",
+    True,
```

Comment:
Default to False until this has been verified to produce sota weights?

Reply:
sg -- Done.